internal/jujuclient/applicationoffers.go

```diff
@@ -209,9 +209,7 @@ func (c Connection) GetApplicationOfferConsumeDetails(ctx context.Context, user
 			OfferURLs:     []string{info.Offer.OfferURL},
 			BakeryVersion: v,
 		},
-		// Do not include a user in the args, Juju will opt to use the user authenticated in the connection.
-		// There is a bug where setting the user tag does not behave as expected.
-		UserTag: "",
+		UserTag: user.String(),
```

Comment:
Can you add a comment about why we do this.

Reply:
done